.changelog/4380.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Added support to **run-specific-validations** flag to the new validate format.
```

Comment:
can we also add a flag to --ignore-specific-validations like we discussed?

Reply:
What would be the point in adding such flag?
I don't think it's necessary